zookeeper-server/src/test/java/org/apache/zookeeper/common/X509UtilTest.java

```diff
@@ -513,4 +518,27 @@ private void setCustomCipherSuites() {
         x509Util.close(); // remember to close old instance before replacing it
         x509Util = new ClientX509Util();
     }
+
+    public static class NullSslContextSupplier implements Supplier<SSLContext> {
+
+        @Override
+        public SSLContext get() {
+            return null;
+        }
+
+    }
+
+    public static class SslContextSupplier implements Supplier<SSLContext> {
+
+        @Override
+        public SSLContext get() {
+            try {
+                return SSLContext.getDefault();
+            } catch (NoSuchAlgorithmException e) {
+                return null;
```

Comment:
nit: `throw new RuntimeException(e);` so if this ever happens (unlikely), the error will be more obvious in the test

Reply:
Will do :+1: